src/index.ts

```diff
@@ -53,6 +53,7 @@ function runScript(script: Script): Promise<void> {
   }
 }
 
-runScript(script).catch(() => {
+runScript(script).catch(error => {
+  logger.bin(`${script} script error: ${error}`);
```

Comment:
does it log it in "red"?
as it's an error I would it to be clearly red in the terminal

Reply:
True, trying to fix that right now

Reply:
Ok, now it's red: 
![image](https://user-images.githubusercontent.com/6418684/50632480-4cad5280-0f48-11e9-9422-addcf576a0b4.png)